projects/serializers/project.py

```diff
@@ -97,9 +97,9 @@ def validate_attribute_data(self, attribute_data):
         return valid_attributes
 
     def create(self, validated_data: dict) -> Project:
-        validated_data["phase"] = ProjectPhase.objects.get(
-            project_type__name="asemakaava", index=0
-        )
+        validated_data["phase"] = ProjectPhase.objects.filter(
+            project_type__name="asemakaava"
+        ).first()
```

Comment:
This is almost my fault since I think I changed this in the importer. :sweat_smile: Though it's also good that it's no longer hard coded here. :+1: